.github/workflows/test-harness-acapy-credo-anoncreds.yml

```diff
@@ -0,0 +1,55 @@
+name: test-harness-acapy-credo-anoncreds
```

Comment:
How are these different from all the other AnonCreds tests we do?  Is it just that ACA-Py is using the “askar-anoncreds” wallet type?   If so, perhaps rename to “acapy-anoncreds-credo”, since AFAIK there are no special Credo settings for the credx vs. anoncreds-rs libraries.

Reply:
There was no test for AnonCred/Revocation (new test added) and I had to tweak a couple of the other AnonCreds tests to meet some of the tag exclusions in the existing Credo runset.

I'll do the other changes you've requested this afternoon.